plugins/snapd-uefi/fu-self-test.c

```diff
@@ -0,0 +1,732 @@
+/*
+ * Copyright 2024 Maciej Borzecki <maciej.borzecki@canonical.com>
```

Comment:
Probably should expand to 2025 and add more authors.

Reply:
I've added Simon (apologies!) but I think the year should remain as the *start* year -- as this code is basically renamed from the old location.